src/fsharp/FSharp.Core/list.fsi

```diff
@@ -693,15 +693,15 @@ namespace Microsoft.FSharp.Collections
         [<CompiledName("Sort")>]
         val sort: list:'T list -> 'T list when 'T : comparison
 
-        /// <summary>Splits a list into two lists, at the given index.</summary>
-        /// <param name="index">The index at which the list is split.</param>
+        /// <summary>Splits a list into two lists, at the first N elements of the list.</summary>
+        /// <param name="count">The number of items to take or skip.</param>
         /// <param name="list">The input list.</param>
         /// <returns>The two split lists.</returns>
         ///
-        /// <exception cref="System.InvalidOperationException">Thrown when split index exceeds the number of elements
+        /// <exception cref="System.InvalidOperationException">Thrown when split count exceeds the number of elements
         /// in the list.</exception>
         [<CompiledName("SplitAt")>]
-        val splitAt: index:int -> list:'T list -> ('T list * 'T list)
+        val splitAt: count:int -> list:'T list -> ('T list * 'T list)
 
```

Comment:
above lines `splitAt`'s parameter `index`, not very exactly,because :
```F#
let res = List.splitAt 3 [1;2;3] // 3 > ls.length - 1. so it is not index.
```
i think `splitAt` equals to take the list and skip the list and than merge their result list to pair tuple. so the doc should consistant to `take` and `skip`.